packages/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain/span_utils.py

```
@@ -50,25 +51,135 @@ def _message_type_to_role(message_type: str) -> str:
 
 
 def _set_span_attribute(span: Span, name: str, value: AttributeValue):
-    if value is not None and value != "":
+    if value is not None and (value != "" if isinstance(value, str) else True):
         span.set_attribute(name, value)
 
 
-def set_request_params(span, kwargs, span_holder: SpanHolder):
-    if not span.is_recording():
-        return
+def _get_unified_unknown_model(class_name: str = None, existing_model: str = None) -> str:
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)